packages/cf-component-dropdown/test/Dropdown.js

```diff
@@ -43,51 +48,57 @@ describe('interactions', () => {
       called = true;
     };
 
-    var dom = <Dropdown onClose={onClose} />;
-    var instance = render(dom, global.root);
+    var dom = mount(felaTestContext(<Dropdown onClose={onClose} />));
+
+    // Fela renders a Stateless component so we can't just use `.getNode()` but
+    // we know that the Dropdown we want is the second element in the list.
+    const instance = dom.find('Dropdown').getNodes()[1];
 
     instance.handleDocumentKeydown({ keyCode: 27 });
     expect(called).toBeTruthy();
   });
 
   test('should be able to navigate with the up and down keys', () => {
-    trackFocus();
-
     const down = { keyCode: 40, preventDefault() {} };
     const up = { keyCode: 38, preventDefault() {} };
 
-    const getFocusedHref = () => getCurrentFocus().getAttribute('href');
-
-    const dom = (
-      <Dropdown onClose={() => {}}>
-        <DropdownLink to="one" />
-        <DropdownLink to="two" />
-        <DropdownLink to="three" />
-      </Dropdown>
+    const getFocusedHref = () =>
+      global.document.activeElement.getAttribute('href');
+
+    const dom = mount(
+      felaTestContext(
+        <Dropdown onClose={() => {}}>
+          <DropdownLink to="one" />
+          <DropdownLink to="two" />
+          <DropdownLink to="three" />
+        </Dropdown>
+      )
     );
-    const instance = render(dom, global.root);
+    const instance = dom.find('Dropdown').getNodes()[1];
+
+    expect(getFocusedHref()).toBe(null);
 
     instance.handleDocumentKeydown(down);
-    expect(getFocusedHref()).toBe('one');
+    setTimeout(() => expect(getFocusedHref()).toBe('one'), 0);
```

Comment:
I could try to do something here but am unsure what that would be. Some sort of callback comes to mind but I ended up doing this and seeing if someone _really_ hates it.